src/views/pages/login/Login.js

```diff
@@ -19,29 +21,48 @@ import { axiosPublic } from 'src/utils/axiosPublic'
 // import { useDispatch, useSelector } from 'react-redux'
 
 const Login = () => {
+  const navigate = useNavigate()
   const [loginLoadingShow, setLoginLoadingShow] = useState(false)
   const [loginErrorMessageShow, setLoginErrorMessageShow] = useState(false)
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
-  const toggleAlert = () => {
-    setLoginErrorMessageShow(!loginErrorMessageShow)
-  }
+  
```

Comment:
Build is failing because of this white space 
<img width="506" alt="image" src="https://user-images.githubusercontent.com/4688675/224144347-ca8ae8d8-cccc-4573-9c10-6a2a929720f8.png">

Reply:
I have fixed this, in the next pr